seeder/mongoDB_seeder/index.js

```diff
@@ -1,6 +1,3 @@
-const seedDBFunc = require('./seedDBFunctions');
-
+const seedDBFunc = require("./seedDBFunctions");
 
 seedDBFunc.setInventoryQuantity(100, 3, seedDBFunc.addSizesToDB);
-
-
```

Comment:
**Annotation:** created new folder mongoDB_seeder to house legacy seeder by Nathan Quiocson